src/config/navigation/defaultStackConfig.tsx

```diff
@@ -255,7 +265,12 @@ export const defaultStackConfig = ({
     },
     {
       routeName: ScreenName.Index,
-      screenComponent: IndexScreen
+      screenComponent: IndexScreen,
+      // NOTE: is used as initial screen for the points of interest tab
```

Comment:
i wonder if we really need this note? what was your intention?

Reply:
This note is not necessary. But I'd still prefer to keep it so that in the future we can understand why it happened the way it did.

Reply:
but what does it help? it says that it is the initial screen for points of interest. yes true thats why it is configured.
or other question, why do we not have NOTEs at the other initial screens? why only for POI? i do not understand.